rllib/evaluation/rollout_worker.py

```diff
@@ -785,12 +783,12 @@ def wrap(env):
                 multiple_episodes_in_batch=pack,
                 normalize_actions=normalize_actions,
                 clip_actions=clip_actions,
-                blackhole_outputs="simulation" in off_policy_estimation_methods,
                 soft_horizon=soft_horizon,
                 no_done_at_end=no_done_at_end,
                 observation_fn=observation_fn,
                 sample_collector_class=policy_config.get("sample_collector"),
                 render=render,
+                blackhole_outputs="simulation" in off_policy_estimation_methods,
```

Comment:
Add TODO to deprecate this once we completely disallow "simulation".

Reply:
what is `blackhole_outputs` ?